src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +79,20 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else
+  {
+    OrionError oe;
+    oe.code = ciP->httpStatusCode;
+    oe.reasonPhrase = "AlreadyExists";
```

Comment:
The "Error response" section in doc/apiary/v2/fiware-ngsiv2-reference.apib document should be edited to add this new error to the list.